### ISSUE TITLE
[BUG] 'NoneType' object has no attribute 'span' new

### ISSUE THREAD

**Issue by bilalfarhan** [pytube/pytube#1225]
Hello
I'm facing a new error now
'NoneType' object has no attribute 'span'
Please contribute to discovering the new changes made by YouTube

**Comment by bilalfarhan** [comment]
About half an hour ago I noticed that
Note: A few hours ago, everything was fine

**Comment by Dheirya** [comment]
I have the same error. Found this https://github.com/pytube/pytube/issues/1218#issuecomment-1028033500 and it fixed it for about 2 hours but back to the same error.

**Comment by bilalfarhan** [comment]
> I have the same error. Found this [#1218 (comment)](https://github.com/pytube/pytube/issues/1218#issuecomment-1028033500) and it fixed it for about 2 hours but back to the same error.

You fixed the bug that appeared two days ago
But this is a new bug that has been discovered

**Comment by Dheirya** [comment]
> > I have the same error. Found this [#1218 (comment)](https://github.com/pytube/pytube/issues/1218#issuecomment-1028033500) and it fixed it for about 2 hours but back to the same error.
> 
> You fixed the bug that appeared two days ago But this is a new bug that has been discovered

No this bug is the same as the bug that appeared two days ago and it seems that the fix from before is not working.

**Comment by bilalfarhan** [comment]
> > > I have the same error. Found this [#1218 (comment)](https://github.com/pytube/pytube/issues/1218#issuecomment-1028033500) and it fixed it for about 2 hours but back to the same error.
> > 
> > 
> > You fixed the bug that appeared two days ago But this is a new bug that has been discovered
> 
> No this bug is the same as the bug that appeared two days ago and it seems that the fix from before is not working.

No, I myself fixed the bug that appeared two days ago and the problem was solved
The new bug was discovered within the past two hours or less

**Comment by juanchosaravia** [comment]
it's not working for me neither but we should close this ticket and continue in the other one

**Comment by bilalfarhan** [comment]
> it's not working for me neither but we should close this ticket and continue in the other one

With all due respect to your point of view
The problem is different and requires a new solution

**Comment by sobotlar** [comment]
> > it's not working for me neither but we should close this ticket and continue in the other one
> 
> With all due respect to your point of view The problem is different and requires a new solution

+

**Comment by a-l-e-k-k** [comment]
any ideas on why the problem is occurring?

**Comment by Bretty01** [comment]
So regarding the issue on hand. I took a gander at the js code again and it appears to be the same issue as stated in https://github.com/pytube/pytube/issues/1218. To put it simply, the temporary fix is to change to `name = "jha"`. This doesn't change that the function used to find the throttler is still unusable because the formatting is still different is it doesn't ask for a function but instead an array; `var Cpa=[jha]` .

Hopefully that works for you all for the time being.

**Comment by a-l-e-k-k** [comment]
It works. For a more permanent solution #1222 looks promising.

**Comment by KelestZ** [comment]
> So regarding the issue on hand. I took a gander at the js code again and it appears to be the same issue as stated in #1218. To put it simply, the temporary fix is to change to `name = "jha"`. This doesn't change that the function used to find the throttler is still unusable because the formatting is still different is it doesn't ask for a function but instead an array; `var Cpa=[jha]` .
> 
> Hopefully that works for you all for the time being.

Doesn't work. I get

KeyError: 'content-length'

**Comment by shivpatil007** [comment]
Any new updates on how to fix this issue !!! its persisting from 2 days

**Comment by ImranRahimov1995** [comment]
> So regarding the issue on hand. I took a gander at the js code again and it appears to be the same issue as stated in #1218. To put it simply, the temporary fix is to change to `name = "jha"`. This doesn't change that the function used to find the throttler is still unusable because the formatting is still different is it doesn't ask for a function but instead an array; `var Cpa=[jha]` .
> 
> Hopefully that works for you all for the time being.

It's working , Thanks

**Comment by sobotlar** [comment]
> > So regarding the issue on hand. I took a gander at the js code again and it appears to be the same issue as stated in #1218. To put it simply, the temporary fix is to change to `name = "jha"`. This doesn't change that the function used to find the throttler is still unusable because the formatting is still different is it doesn't ask for a function but instead an array; `var Cpa=[jha]` .
> > Hopefully that works for you all for the time being.
> 
> It's working , Thanks

no working

**Comment by ImranRahimov1995** [comment]
> > > So regarding the issue on hand. I took a gander at the js code again and it appears to be the same issue as stated in #1218. To put it simply, the temporary fix is to change to `name = "jha"`. This doesn't change that the function used to find the throttler is still unusable because the formatting is still different is it doesn't ask for a function but instead an array; `var Cpa=[jha]` .
> > > Hopefully that works for you all for the time being.
> > 
> > 
> > It's working , Thanks
> 
> no working

Actually working .
I have Django-app based on pytube

https://youtube.impropy.me/

if something going wrong ,app has ServerError500
maybe you make something wrong .

**Comment by MYtelecharger** [comment]
Is anyone fix this problem?

**Comment by glubsy** [comment]
This is a duplicate, and the fix is at https://github.com/pytube/pytube/pull/1222

**Comment by Bretty01** [comment]
> This is a duplicate, and the fix is at #1222

Yes, use this pull request for a better fix. Otherwise you guys will probably be coming back here on a daily basis complaining about your code not working due to the same issue lol.

**Comment by DEADF00D** [comment]
The issue is on the regex of the `func_regex` of `parser.throttling_array_split()` and on the regex `function_patterns[0]` of `cipher.get_throttling_function_name()`  
Published an according fork for this fix.  
Succesfully tested, now in production on my app.
